### DISCUSSION TITLE
any project in production stage?

### DISCUSSION THREAD

**Discussion by mmmaxxxe** [directus/directus#15906] [Archive]
Hi 
i started to develop an easy application using directus but i feel a little bit frustrated. After any upgrade, something that was stable is broken, new bugs, emerging bugs, old bugs come back....

I'm really trying to understand if directus is a real development platform or if it's just for fun.

**Comment by benhaynes** [comment]
I mean, yes, we're doing this for fun in the sense that we love building it. But more seriously, this is a "real" project. You aren't giving specifics, so I can't speak to those... but what I can say is that we are actively working on improving our testing and release process to avoid regressions. Like any software, there will be bugs... and we're trying to keep those to a minimum, but that's about all I can say about this without more info.
